awx/ui/test/e2e/fixtures.js

```diff
@@ -10,6 +10,15 @@ import {
 const session = `e2e-${uuid().substr(0, 8)}`;
 const store = {};
 
+/* Utility function for accessing awx resources. This includes resources like
+ * users, organizations, and job templates. Creates the resource if it does
+ * not already exist, then returns it.
+ *
+ * @param endpoint - The REST API url suffix.
+ * @param data - Attributes used to create a new endpoint.
+ * @param [unique=['name']] - A unique key for a resource.
```

Comment:
Perhaps "An array of keys used to uniquely identify previously created resources from the endpoint. " ?

Reply:
(just as an aside - not for this PR) This function really does do _quite_ a lot and should probably be refactored into several smaller functions when time permits.